packages/core/core/src/Parcel.js

```diff
@@ -195,4 +200,24 @@ export default class Parcel {
   package(bundles: any[]) {
     return Promise.all(bundles.map(bundle => this.runPackage(bundle)));
   }
+
+  handleChange(filePath: string) {
```

Comment:
Can we move this logic into the `AssetGraph`? That way the `Parcel` class stays mostly unaware of the graph beyond telling it that a file has been transformed, a file has been resolved, or a file has changed.